packages/feedback-spa/src/mocks/options.js

```diff
@@ -0,0 +1,26 @@
+export const options = [
+  {
+    id: 0,
+    name: 'Mod Handovers'
+  },
+  {
+    id: 1,
+    name: 'Outages'
+  },
+  {
+    id: 2,
+    name: 'DSAL'
+  },
+  {
+    id: 3,
+    name: 'Feedback'
+  },
+  {
+    id: 4,
+    name: 'Projects & Initiatives'
+  },
+  {
+    id: 5,
+    name: 'Service Status'
+  }
+];
```

Comment:
Newline at the end of file.

```suggestion
];

```